### DISCUSSION TITLE
Peridynamics with Phase Field Equations

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#19536] [Q&A General] [answered]
Dear MOOSE Community,

I wish to use phase field equations along with peridynamics constitutive model for which I am concerned whether the peridynamic solution is redistributed to Finite Element nodes so that they can be utilised in phase field equations or one has to develop a corresponding constitutive model in peridynamics which mimicks evolution of phase field.Any headers in this regard will be beneficial.

Many Thanks
Pushkar

**Comment by jiangwen84** [comment, marked as answer]
It is not clear to me what you want to do here. Maybe it is better to write down your algorithms. 

In peridynamics, line elements are used to model the bonds. The phase field will be solved on a standard element meshes. They use different meshes, so I do not believe there is a simple way to use the peridynamics solution directly in phase field. However, it might be possible to use Multi-App system that solves phase-field and peridynamics separately and then transfer the data between two simulation.

**Reply by ppandit95** [reply]
ohh thanks @jiangwen84 for suggesting possible ways to integrate peridynamics solution with phase field evolution...
